### ISSUE TITLE
Modify response headers not working

### ISSUE THREAD

**Issue by krzaku12** [requestly/chrome-extension#127]
The specific header I need to change is content security policy, but I also tried others just to test. I tried with a specific URL as well as blank URL (meaning all sites). Also tested in incognito mode with all other extensions disabled to see if there's a conflict but the outcome is the same. Request header modification works fine though.

Chrome version: 50.0.2661.102
Requestly version: 4.1.3

**Comment by blunderboy** [comment]
@krzaku12 Probably a rule setup issue. I tried to Remove Response Header - `X-Frame-Options` and it worked fine in Chrome v50. Here is a simple way to test.

1. Open this URL: http://jsbin.com/jomojawate/1/edit?html,output
2. Confirm that yahoo.com sets X-Frame-Options header which does not allow it to render in Iframe. 
3. Created a rule to remove X-Frame-Options header for Urls containing yahoo. Here is a snapshot
<img width="1145" alt="screenshot 2016-05-22 10 08 49" src="https://cloud.githubusercontent.com/assets/3108399/15452239/5b1b6e40-2005-11e6-8704-bdbe9cbf5240.png">
4. Confirm that site opens fine in Jsbin now. Here is a screenshot
<img width="1268" alt="screenshot 2016-05-22 10 10 34" src="https://cloud.githubusercontent.com/assets/3108399/15452242/72895812-2005-11e6-9ea4-6c7fc1501a52.png">

 Can you tell what are the steps you are trying and show me your rule, I will try to help.

**Comment by krzaku12** [comment]
Correct, it loads the frame, though the header is not changed in chrome network inspector which is what I was looking at, sorry for the confusion.

I didn't look at the error in console close enough to notice it changed after adding the rule and now it mentioned `Access-Control-Allow-Origin` header. After modifying this header everything works as expected.

Would it be possible to add a note somewhere mentioning the actual header in network inspector doesn't change? So people like me don't look for a problem where there isn't one.

**Comment by blunderboy** [comment]
@krzaku12 Thanks for confirming. I added an alert box earlier in Headers rule specifying

> Modifications in Response Headers are not visible in Chrome Dev Tools.

I removed this intentionally. I will put it back as part of this issue.
